app/templates/main/serviceTranscript.html

```diff
@@ -0,0 +1,110 @@
+{% extends "base.html"%}
+
+
+{% block scripts %}
+  {{super()}}
+
+
+{% endblock %}
+
+{% block styles %}
+  {{super()}}
+
+{% endblock %}
+
+
+{% block app_content %}
+
+<h1 class="text-center pb-5">{{g.current_user.firstName}} {{g.current_user.lastName}}</h1>

```

Comment:
Can't use the current-user here, or admins won't be able to print other people's transcripts